### ISSUE TITLE
Сравнение пользователей выполняется очень долго и градиент цветов не зависит от числа общих друзей

### ISSUE THREAD

**Issue by disovi** [ElCapitanos/vk-api-test#1]
![image](https://user-images.githubusercontent.com/705617/174287227-2ae82985-1f15-4e4a-b4ca-8b2047232da6.png)

при этом задача работала минут 5 и ела кучу CPU, хотя требует всего 2х запросов.

**Comment by ElCapitanos** [comment]
Да, там проблема с кол-вом запросов в самом ВК, казалось, что решил, но, по
всей видимости, нет (

пт, 17 июн. 2022 г., 14:10 Vladislav ***@***.***>:
